app/views/account/comments/_comment.html.slim

```diff
@@ -0,0 +1,13 @@
+.comment-list id="comment_#{comment.id}"
+  - if current_user.privileged?
+    = link_to account_comment_path(comment), :method => :delete,
+                       remote: true, data: { confirm: 'Are you certain you want to delete this comment?'},
+                       class: 'close' do
+      i.fa.fa-close
+  span.avatar-img = gravatar_image_tag(current_user.email, gravatar: { size: 50 })
+  .media-body
+    h4.media-heading.user_name
+      = link_to comment.user.full_name, account_user_path(comment.user)
+  p = comment.body
```

Comment:
use simple_format and remove "p"

Reply:
feels like `p` is out of lvl..
why do you need 
```
.media-body
  h4.media-heading.user_name
    = link_to comment.user.full_name, account_user_path(comment.user)
```
but `p` is on the same lvl as `.media-body`
in other words, why do you need `.media-body` container only for title?